tensornetwork.py

```diff
@@ -521,60 +523,59 @@ def _remove_trace_edge(self, edge: Edge, new_node: Node) -> None:
       new_node.add_edge(e, i)
     self.nodes_set.remove(edge.node1)
 
-  def _remove_edge(self, edge: Edge, new_node: Node) -> None:
-    """Collapse an edge in the network.
+  def _remove_edges(self, edges, node1: Node, node2: Node, new_node: Node) -> None:
+    """Collapse a list of edges shared by two nodes in the network.
```

Comment:
Passing the edges and the nodes is redundant. Just pass the edges

Reply:
Actually, it's so that the ordering of the arguments to tensordot can be specified. Previously, the ordering was `edge.node1, edge.node2`. Now we have multiple edges, so there's no unique choice based on them. Instead, we use the arguments to e.g. `contract_between` to decide the ordering. This data must be passed on to `reorder_edges`, which needs to know which axes of `new_node` correspond to which axes of the old nodes.

Also, one may wish to choose the ordering to minimize the need for transposes of the output tensor.